README.md

```diff
@@ -1,6 +1,12 @@
 # AIND Metadata Entry
+User-interface to ease AIND metadata ingest and validation. 
```

Comment:
I thought we were also gonna add message on the app page?

Reply:
whoops i forgot, just added it looks like this: 
![Screen Shot 2023-06-29 at 3 47 44 PM](https://github.com/AllenNeuralDynamics/aind-metadata-entry-js/assets/54870020/30cbca0a-70de-4923-a5f6-06a4aecc9415)